.gitignore

```diff
@@ -1,2 +1,3 @@
+node_modules
```

Comment:
coc.nvim doesn't want node_modules to join the .gitignore file

Reply:
I use npm more for development but ok, throw it as it was :)